pkg/morph/client/notifications.go

```diff
@@ -131,7 +131,7 @@ func (c *Client) ReceiveNotaryRequests(txSigner util.Uint160) error {
 		return nil
 	}
 
-	_, err := c.client.ReceiveNotaryRequests(&neorpc.TxFilter{Signer: &txSigner}, c.subs.curNotaryChan)
+	_, err := c.client.ReceiveNotaryRequests(&neorpc.NotaryRequestFilter{Signer: &txSigner}, c.subs.curNotaryChan)
```

Comment:
As you see, 0.105.0 introduces the ability to filter out notary requests by mempool event type. IINM, you need to update node's code to subscribe only for `added` notary request, because node doesn't care about `removed` requests.

Reply:
fixed